### ISSUE TITLE
[7.4.8] Problem with the Lint (and Android version)

### ISSUE THREAD

**Issue by jonnymelabo** [renpy/renpy#3031]
```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/front_page.rpy", line 271, in <module>
  File "game/front_page.rpy", line 281, in _execute_python_hide
TypeError: environment can only contain strings

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "launcher/game/front_page.rpyc", line 271, in script
  File "C:\Renpy\renpy-6.99.12.4-sdk/renpy/ast.py", line 923, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Renpy\renpy-6.99.12.4-sdk/renpy/python.py", line 2235, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/front_page.rpy", line 271, in <module>
  File "game/front_page.rpy", line 281, in _execute_python_hide
  File "C:\Renpy\renpy-6.99.12.4-sdk\editra\Editra.edit.py", line 124, in end
    self.launch_editra()
  File "C:\Renpy\renpy-6.99.12.4-sdk\editra\Editra.edit.py", line 110, in launch_editra
    ], cwd=DIR, env=env)
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python2.7/subprocess.py", line 325, in init
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python2.7/subprocess.py", line 575, in _execute_child
TypeError: environment can only contain strings

Windows-10-10.0.19041
Ren'Py 7.4.8.1895
Ren'Py Launcher 7.4.8.1895
Sat Sep 11 17:55:55 2021
```
Here is the lint file found in the renpy folder:
[lint.txt](https://github.com/renpy/renpy/files/7148193/lint.txt)

It's been a while since I can't make the Android version of the game (around 6 months).

One of the problems that went commented in the lint was a docx file, but itw as in a folder that it was in theory ignored by renpy using this:

`build.classify('game/images/startmenu/promotion/**', None)`

Since the message of a friend told me this could be the issue:

```
images/startmenu/promotion/CorrecionTEXTO/2020_Leropia/Diálogo_Letropía.docx contains non-ASCII characters in its filename.
(ZIP file distributions can only reliably include ASCII filenames.)
```

I even moved the whole folder to somewhere else trying to see  if that fixed the problem (since my friend using the files from the build itself was able to do the android version, but I can't).

I just don't know what else to do to make the lint work or just the android version work.

Thanks!

**Comment by renpytom** [comment]
Editra isn't a supported text editor, anymore.

**Comment by jonnymelabo** [comment]
@renpytom

Now I used Atom and at least it gived me more information, still an error... but There's some more information.

```
I'm using Gradle to build the package.
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.1.1/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> android.defaultConfig.versionCode is set to 0, but it should be a positive integer.
  See https://developer.android.com/studio/publish/versioning#appversioning for more information.

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

[AndroidBroke.txt](https://github.com/renpy/renpy/files/7148216/AndroidBroke.txt)

**Comment by jonnymelabo** [comment]
Ok problem solved.

Two things were causing this problem.

1. I was still using Editra.

2. I was using the version 0.0.0 (which being 0 was giving an error) Later I changed the number, but for some reason the launcher was still remembering that 0. so I deleted RAPT and reinstalled it with a random number (0.4.6) And it finally worked.

I hope now it will work for a long time before having a new issue with Android again. ^^
